project/plugins.sbt

```diff
@@ -2,5 +2,5 @@ resolvers += Resolver.url(
   "lila-maven-sbt",
   url("https://raw.githubusercontent.com/ornicar/lila-maven/master")
 )(Resolver.ivyStylePatterns)
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"   % "2.8.18")
```

Comment:
I didn't intend that to be used outside lila itself, but ok... I guess it's necessary to work around the scala-xml dependency mess